### DISCUSSION TITLE
Image annotations with Plotly

### DISCUSSION THREAD

**Discussion by ryanchesler** [h2oai/wave#670] [Q&A]
Is there any way to have Plotly information sent back to the wave app? Plotly is able to trigger some interaction and send some information to Dash, but I am not sure if I would be able to output a HTML plot like shown in the example here and then draw some annotations on the plotly UI and then feed that information back to wave in some way. 

https://dash.plotly.com/annotations

It would be powerful to be able to do something like the section that shows "Drag and draw annotations - use the modebar to pick a different drawing tool"

**Comment by lo5** [comment]
Related: #313 

We'll look at a more general way to incorporate third party javascript libraries via #297.
